pkg/kyma/docs/install.md

```diff
@@ -0,0 +1,108 @@
+# install
+
+## Description
+
+Use this command to install Kyma on a running Kubernetes cluster.
+
+## Usage
+
+```
+kyma install [OPTIONS]
+```
+
+## Options
+
+| Name                    | Short Name | Default value| Description|
+| :----------------------:|:---------:|:-----:|------|
+| **&#x2011;&#x2011;release**  | -r ||Specifies the Kyma release or git revision to be installed. Go to the [GitHub releases page](https://github.com/kyma-project/kyma/releases) to find out more about each of the available releases, or use the revision of your choice. For example, write `kyma install --release master`.|
+| **&#x2011;&#x2011;local**    | -l |`false`|Indicates local installation using Kyma sources. If the location of your cloned  `kyma-cli` repository follows the Go code conventions, the CLI finds it automatically. If not, you must configure the path explicitly using `--src-path`.| 
+| **&#x2011;&#x2011;config**   | -c ||Specifies the URL or path to the Installer configuration yaml file.| 
+| **&#x2011;&#x2011;override** | -o ||Specifies the path to a yaml file with parameters to override.| 
+| **&#x2011;&#x2011;domain**   | -d |`kyma.local`|Specifies the domain used for installation.| 
+| **&#x2011;&#x2011;password** | -p ||Specifies the predefined cluster password.| 
+| **&#x2011;&#x2011;noWait**   | -n |`false`|Determines if the installation should wait for the Installer configuration to complete.| 
```

Comment:
```suggestion
| **&#x2011;&#x2011;noWait**   | -n |`false`|Determines if the installation should wait for the Kyma Operator configuration to complete.| 
```

Reply:
Determines if the installation command should wait for the Kyma installation to complete.